factory/factory.go

```diff
@@ -52,12 +51,8 @@ func InitConfigFactory(f string) error {
 		if UdrConfig.Configuration.WebuiUri == "" {
 			UdrConfig.Configuration.WebuiUri = "webui:9876"
 		}
-		roc := os.Getenv("MANAGED_BY_CONFIG_POD")
-		if roc == "true" {
+		if os.Getenv("MANAGED_BY_CONFIG_POD") == "true" {
```

Comment:
Should this be changed to "false" and move the `else` code into the `if`... Something like shown below:
```suggestion
		if os.Getenv("MANAGED_BY_CONFIG_POD") == "false" {
			go func() {
				logger.InitLog.Infoln("use helm chart config")
				ConfigPodTrigger <- true
			}()
		}
```
Also, I do not see this code in the `nrf`. Do you know why this difference?

Reply:
Yes, I know why NRF does not have it. NRF updateConfig method does not use any channel (ConfigPodTrigger). Hence, if we use a similar if else clause in NRF,  there is nothing to do.

Reply:
The code is moved under service/init.go.